MesaModel/hamilton/reframe_parameterisation_infrastructure/reframe_tests/parameterisation_tests.py

```diff
@@ -1,26 +1,23 @@
 import os
+import csv
 
 import reframe as rfm
 import reframe.utility.sanity as sn
 
+with open("../../lhs_sampling/lhs_params.csv", "r") as f:
+    csv_reader = csv.reader(f)
+    ROWS = [",".join(row[1:]) for row in list(csv_reader)]
```

Comment:
It should be a hard requirement on the CSV to make test ids correspond to index otherwise the names of the tests assigned by ReFrame will be wrong. I could modify line 9 to add a check for this? Is this necessary?

Reply:
I think a check would be worthwhile to enforce the requirement - it might save issues later on if e.g. a line was deleted from a CSV so the IDs didn't line up.